.gitignore

```diff
@@ -0,0 +1,2 @@
+
+assets-wp-repo/Thumbs.db
```

Comment:
This .gitignore file can be removed from the pr

Reply:
ok, both done with latest commit.